server/utils/assessments/utils.ts

```diff
@@ -334,7 +334,8 @@ const daysUntilDue = (assessment: Assessment): number => {
 
 const getStatus = (assessment: Assessment): string => {
   if (assessment.status === 'completed') {
-    return `<strong class="govuk-tag govuk-tag">Completed</strong>`
+    if (assessment.decision === 'accepted') return `<strong class="govuk-tag govuk-tag--green">Suitable</strong>`
```

Comment:
Is "Suitable" definitely the wording we want here?

Reply:
Beth says she hasn't been able to go through this with Selina yet so the colours and copy are provisional for now.

Reply:
Suitable was suggested in Slack: https://mojdt.slack.com/archives/C02SHKETZ7F/p1676460188359239 
But I think Accepted might make more sense

Reply:
No drama - let's go with as is 👍